src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/jclouds/aws/AWSEC2JCloudsProvider.java

```diff
@@ -243,7 +243,7 @@ public String addToInstancePublicIp(Infrastructure infrastructure, String instan
             try {
                 elasticIPAddressApi.associateAddressInRegion(region, unassignedIp.getPublicIp(), id);
             } catch (RuntimeException e) {
-                continue;
+                logger.warn("Cannot associate address " + unassignedIp.getPublicIp() + " in region " + region, e);
             }
```

Comment:
by removing continue, you will change the logic. Maybe it's ok, but need to be checked